src/main/java/io/qameta/allure/bamboo/AllureConstants.java

```diff
@@ -3,23 +3,45 @@
 /**
  * The internal class with some constants needed to Allure plugin.
  */
-final class AllureConstants {
+public interface AllureConstants {
```

Comment:
Bad practice is to expose constants in general, much better to have them private or at least to have getters for them. But unlike Bloch, I hate verbosity, which is meant in having final class with private constructor and public static modifiers for every constant.
If we don't expose the interface (let's make it package private), it's not an anti-pattern.